.changeset/witty-kids-repair.md

```diff
@@ -0,0 +1,6 @@
+---
+'@cloudflare/next-on-pages-next-dev': patch
```

Comment:
```suggestion
```

I don't think we should include change logs for `next-on-pages-next-dev` as it is not a package that the user sees anyways, that's my bad as I didn't exclude it from changesets, I'll do that now, could you remove this anyways? 🙂